README.md

```diff
@@ -60,7 +60,7 @@ It is setup with two primary machines: web and db. Together these two virtual ma
 4. Create a Magento 2 build available at m2.demo:
 
     ```bash
-    vagrant ssh web -- /server/vagrant/bin/m2setup.sh --sampledata --hostname=m2.demo
+    vagrant ssh web -- /server/vagrant/guest/bin/m2setup.sh --sampledata --hostname=m2.demo
```

Comment:
You don't need to update your path, because the PATH already contains a pointer to /usr/local/bin on the guest, allowing the script to be run directly _inside the vagrant machine_. The full command is run from the host machine, which is why `vagrant ssh web -- ` prefixes the command; it runs what follows it inside the guest machine. If you ran simply `m2setup.sh` inside the guest it would work as well.

Thanks for updating!